lib/smartSurveyAPICall.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client')
+
+module.exports = {
+  getResponses: (baseUrl, surveyID, endPoint, token, tokenSecret) => {
+    querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' });
+    const apiUrlPart = querystring.stringify({'api_token':token, 'api_token_secret': tokenSecret});
+    axios.get(`${baseUrl}/${surveyID}/${endPoint}?${apiUrlPart}`)
+      .then(function (response) {
+        console.log(response);
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+  },
+
+  createClient: function (token, tokenSecret) {
+    this.client = new SmartSurveyClient({ apiToken: token, apiTokenSecret: tokenSecret})
+  },
```

Comment:
Inconsistent spacing here. You could go for the 
```
module.exports {
  getResponses: getResponses,
  createClient: createClient
  ...
}
```
which might tidy it up a little